syft/__init__.py

```diff
@@ -18,6 +18,7 @@
 from syft.workers import VirtualWorker
 from syft.workers import Plan
 from syft.workers import func2plan
+from syft.workers import meth2plan
```

Comment:
Can we change the name to not be meth2plan, more something like method2plan (certain connotations in english)

Reply:
Hahaha yes you're right!